database-commons/src/main/java/io/cdap/plugin/db/batch/config/DatabaseConnectionConfig.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.db.batch.config;
+
+import java.util.Map;
+
+/**
+ * Interface for DB connection config
+ */
+public interface DatabaseConnectionConfig {
+
+  boolean containsMacro(String property);
```

Comment:
I don't think you need this method, this method is available in `PluginConfig`, this interface shouldn't need it to compute macro

Reply:
discussed offline, did a refactoring.